### PR TITLE
🌱 Set `DefaultClusterSet` default value to true in feature gate

### PR DESCRIPTION
**Pull request by RokibulHasan7**
## Related issue(s)

Fixes #

### CODE REVIEW

File: feature/feature.go [hunk 1]
```diff
@@ -96,7 +96,7 @@ var DefaultSpokeRegistrationFeatureGates = map[featuregate.Feature]featuregate.F
 // feature keys for registration hub controller.  To add a new feature, define a key for it above and
 // add it here.
 var DefaultHubRegistrationFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
-	DefaultClusterSet:          {Default: false, PreRelease: featuregate.Alpha},
+	DefaultClusterSet:          {Default: true, PreRelease: featuregate.Alpha},
```
Comment (qiujian16) [inline feature/feature.go:99]: commonly alpha feature should be false by default, we would need to promote it to beta feature.

Comment (RokibulHasan7) [reply feature/feature.go:99]: I see AddonManagement is also an alpha feature, and it's set to True. any plans to make it beta?

Comment (qiujian16) [reply feature/feature.go:99]: hrm, it is not desired to be still in alpha, it should also be promoted to beta.

Comment (RokibulHasan7) [reply feature/feature.go:99]: Should I promote both to beta?

Comment (qiujian16) [reply feature/feature.go:99]: Think about it twice. Any reason why you want to enable it by default, do you want to always create the default clusterset or global clusterset?

Comment (RokibulHasan7) [reply feature/feature.go:99]: We're enabling the DefaultClusterSet from [here](https://github.com/open-cluster-management-io/ocm/blob/7fd733082f5cf4de1af969b2edc6da1c07f14904/pkg/operator/helpers/helpers.go#L76) 
So, I think we can set Default true.

Comment (qiujian16) [reply feature/feature.go:99]: hrm that makes sense. then after this is enabled by false, i think we can remove the code in https://github.com/open-cluster-management-io/ocm/blob/7fd733082f5cf4de1af969b2edc6da1c07f14904/pkg/operator/helpers/helpers.go#L76

**Pull request merged**